third-party/realdds/include/realdds/topics/control/control-msg.h

```diff
@@ -22,31 +22,31 @@ class control
 #pragma pack( push, 1 )
     enum class control_type : uint16_t
     {
-        SENSOR_OPEN,
-        SENSOR_CLOSE,
+        PROFILES_OPEN,
```

Comment:
How will this work?
If we want 2 profiles opened together? IR + Depth?
We will have 2 control messages right? and the server will have to deal with both?
Not sure it can work on his side, did you check?

Reply:
Sensor has API to open one or multiple streams.
We will send one control message with all the requested streams to open.
The server will transform it to an appropriate API call for each sensor involved, i.e. will have to trace sensor based on stream unique id.